MetaMask/metamask-mobile
# feat: trigger E2E from PR label

## **Description**
This PR enables E2E to run on every PR push that is out of the draft stage. It is an effort to reduce friction for E2E.

## **Manual testing steps**

_1. Create a PR and push a commit message without "e2e" see bitrise step fail
_1. Create a PR and push a commit message with "e2e" see bitrise step succeed and start tests

## **Screenshots/Recordings**

[Fail](https://app.bitrise.io/build/3bb26e49-4416-43ec-82d1-b22f71af349c)

[Success](https://app.bitrise.io/build/3a4c0382-0a2d-4a5c-8573-3473ac30c231)

_Fixes #???_

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
